src/VisualStudio/Core/Def/Implementation/Options/LanguageSettingsPersister.cs

```diff
@@ -33,7 +33,14 @@ internal sealed class LanguageSettingsPersister : ForegroundThreadAffinitizedObj
 
         private readonly IComEventSink _textManagerEvents2Sink;
 
-        private readonly IBidirectionalMap<string, Guid> _languageMap;
+        /// <summary>
+        /// The mapping between language names and Visual Studio language service GUIDs.
+        /// </summary>
+        /// <remarks>
+        /// This is a map between string and <see cref="Tuple{Guid}"/> rather than just to <see cref="Guid"/>
```

Comment:
Speaking with @KevinH-MS that was implied not to be the case. I most definitely saw JIT here in our traces.